### ISSUE TITLE
Clicking on the version drop-down menu in the handbook returns to the introduction

### ISSUE THREAD

**Issue by cmnrd** [lf-lang/lf-lang.github.io#235]
I intuitively click on the little down arrow next to the version selection drop-down menu. Apparently, it is sufficient to hover over the menu and then select a version. However, my brain seems to be hard-coded to clicking on drop-downs. This in itself is not a problem, but clicking on the drop-down menu actually navigates back to the introduction page, which breaks my flow. Is there a way to avoid navigating to another page when clicking on the menu?

**Comment by axmmisaka** [comment]
I'll look for configs, if one does not exist I'll try to fix it and file a PR for docusaurus repo

**Comment by axmmisaka** [comment]
I do not think I could reproduce the problem, unless you click version number in a non-document page. Do you think it would be better if it does nothing or do not appear when not on docs page?

**Comment by cmnrd** [comment]
I only noticed your response now... I think it would be best to do nothing when clicking on the current version (or the arrow next to it).

What do you mean by 'non-document' page? For me, it quite reliably jumps to the introduction page of the handbook when I am clicking on the version in any handbook page that is not the introduction page.

**Comment by axmmisaka** [comment]
https://github.com/lf-lang/lf-lang.github.io/assets/6500159/db81b34f-2a25-4214-b61e-305cde79f9f2

This is the behaviour on my end - can you check if it's the same on your end?

**Comment by cmnrd** [comment]
I think there is still a misunderstanding about the issue. The version dropdown menu works fine, and if I select a version, it stays on the same page. However, my issue is that I am used to clicking on the little arrow that appears next to the current version in order to open a dropdown menu. This is not needed in our case, as the menu opens when hovering over the current version. But if I don't think about it, my brain just lets me click on the current version to open the menu, and when I do that, it jumps back to the introduction page. 

tl;dr When I click on the current version (not the dropdown), then it jumps to the introduction page, which is weird.

**Comment by axmmisaka** [comment]
> I think there is still a misunderstanding about the issue. The version dropdown menu works fine, and if I select a version, it stays on the same page. However, my issue is that I am used to clicking on the little arrow that appears next to the current version in order to open a dropdown menu. This is not needed in our case, as the menu opens when hovering over the current version. But if I don't think about it, my brain just lets me click on the current version to open the menu, and when I do that, it jumps back to the introduction page.
> 
> tl;dr When I click on the current version (not the dropdown), then it jumps to the introduction page, which is weird.



> I think there is still a misunderstanding about the issue. The version dropdown menu works fine, and if I select a version, it stays on the same page. However, my issue is that I am used to clicking on the little arrow that appears next to the current version in order to open a dropdown menu. This is not needed in our case, as the menu opens when hovering over the current version. But if I don't think about it, my brain just lets me click on the current version to open the menu, and when I do that, it jumps back to the introduction page.
> 
> tl;dr When I click on the current version (not the dropdown), then it jumps to the introduction page, which is weird.

Got it! This is counterintuitive and I think an upstream fix would be the most proper way of doing it (i.e. I'll PR in docusaurus repo)

**Comment by cmnrd** [comment]
Great. Thank you!

**Comment by axmmisaka** [comment]
This is closed after the great contributors of docusaurus wrote https://github.com/facebook/docusaurus/pull/10288

**Comment by cmnrd** [comment]
Cool! How do we make this available in our builds, though?

**Comment by axmmisaka** [comment]
> Cool! How do we make this available in our builds, though?

After upstream (docusaurus) releases a new version with this feature, we update our `package.json` - so it's just a matter of time, and we need to update eventually (I plan to have dependabot enabled)
